.cirrus.yml

```diff
@@ -927,8 +922,16 @@ rootless_system_test_task:
 minikube_test_task:
     name: *std_name_fmt
     alias: minikube_test
-    # Docs: ./contrib/cirrus/CIModes.md
-    only_if: *not_tag_magic
+    # Docs: ./contrib/cirrus/CIModes.md (Cirrus Task contexts and runtime modes)
+    # only when: - main rules (see doc above); or
+    #            - minikube test code is changed or one of the shared helper import files from  the system test; or
```

Comment:
extra space between "from" and "the" (I am really getting nitpicky aren't I). Same below.